dbt/adapters/athena/impl.py

```diff
@@ -810,11 +812,12 @@ def persist_docs_to_glue(
             glue_client = client.session.client("glue", region_name=client.region_name, config=get_boto3_config())
 
         # By default, there is no need to update Glue Table
-        need_udpate_table = False
+        need_to_update_table = False
         # Get Table from Glue
         table = glue_client.get_table(CatalogId=catalog_id, DatabaseName=relation.schema, Name=relation.name)["Table"]
         # Prepare new version of Glue Table picking up significant fields
-        updated_table = self._get_table_input(table)
+        table_input = self._get_table_input(table)
+        table_parameters: Dict[str, str] = dict(table_input["Parameters"])
```

Comment:
I think this conversion is not needed because `_get_table_input` returns a type from mypy_boto3 no and `Pameters` is already a `Mapping[str, str]`